components/ospcommon/library.cpp

```diff
@@ -29,6 +29,50 @@
 #include <sys/times.h>
 #endif
 
+extern "C" {
+
+  /* Export a symbol to ask the dynamic loader about in order to locate this
+   * library at runtime. */
+  OSPCOMMON_INTERFACE const int _ospray_anchor()
```

Comment:
The `const` here is leftover from previous iterations on this function. It should be removed. Want a PR for it or is it easier for you to just directly commit the change?

Reply:
Already removed it locally in ‘release-1.8.x’. :)